### PR TITLE
Remove save a page

### PR DESCRIPTION
**Pull request by huwd**
Remove the SavedPages feature

We are reverting [Save a page #69][1] from release_203. The feature was
only ever deployed to integration for testing. We have since decided to
go in a different direction and will replace this with a different
feature to meet the user need.

[Trello](https://trello.com/c/8j4lnc0Y/1049-remove-savedpages-feature)
[1]: #69

## See also

[gds-api-adpters#1108](https://github.com/alphagov/gds-api-adapters/pull/1108)
[gds-api-adpters#1109](https://github.com/alphagov/gds-api-adapters/pull/1109)
[government-frontend#2230](https://github.com/alphagov/government-frontend/pull/2230)

### CODE REVIEW

File: config/content_items.yml [hunk 1]
```diff
@@ -16,11 +16,3 @@ special_routes:
...
     title: "Account home page"
     rendering_app: "frontend"
-  - content_id: "e5098fc2-ad79-4c6f-882b-410831c12f60"
-    base_path: "/account/saved-pages/add"
-    title: "Save a page"
-    rendering_app: "frontend"
-  - content_id: "3aeaeb46-b3c5-449b-8a3b-2753ffef3c9f"
-    base_path: "/account/saved-pages/remove"
-    title: "Remove a saved page"
-    rendering_app: "frontend"
```
Comment (barrucadu) [inline config/content_items.yml:26]: We'll want to unpublish these content items

Comment (huwd) [reply config/content_items.yml:26]: Yeah was going to ask about those - This is just a draft for now until the tests pass

**Comment by barrucadu** [comment]
I think you can also remove `app/validators/absolute_path_validator.rb` in this PR, as it was just used for saved pages.

File: config/content_items.yml (hunk 1 above)
Comment (huwd) [reply config/content_items.yml:26]: Cool this is now ready for a proper look, will confirm here when i've unpublished in all 3 envs

**Review by barrucadu — COMMENTED** [review]
I'll approve this once the pages have been unpublished and the gds-api-adapters change has propagated into government-frontend and frontend (which are the only things calling these endpoints)

File: config/content_items.yml (hunk 1 above)
Comment (barrucadu) [reply config/content_items.yml:26]: I've unpublished these as I needed to publish the special route for `/sign-in/redirect`

**Comment by huwd** [comment]
Ok I've unpublished both those two content IDs across all three environments

**Comment by huwd** [comment]
These three are now all merged: 🎉 
- [gds-api-adpters#1108](https://github.com/alphagov/gds-api-adapters/pull/1108)
- [gds-api-adpters#1109](https://github.com/alphagov/gds-api-adapters/pull/1109)
- [government-frontend#2230](https://github.com/alphagov/government-frontend/pull/2230)

**Comment by barrucadu** [comment]
Can you also do https://github.com/alphagov/frontend/pull/2952 ?

**Comment by huwd** [comment]
On it!

**Comment by huwd** [comment]
https://github.com/alphagov/frontend/pull/2956

**Review by barrucadu — APPROVED** [review]

**Pull request merged**